use-cases.md

```diff
@@ -0,0 +1,150 @@
+---
+layout: page
+title: Instrumentation for Common Use Cases
+---
+<div id="toc"></div>
+
+This page aims to illustrate the common use cases that developers who instrument their applications with OpenTracing API need to deal with.
```

Comment:
nit: s/the common/common/, s/their applications/their applications and libraries/ ?